doc/cephadm/host-management.rst

```diff
@@ -397,6 +397,62 @@ run on the host.
   profiles with the same name are not overwritten.
 
 
+Custom Logrotate Configs
+========================
+
+Cephadm can write out custom logrotate configs for either the
+cephadm.log or the cluster logs to all hosts in the cluster. It's
+recommended to take the existing logrotate file and then add fields
```

Comment:
How?

Also, "It's recommended" strikes me as odd wording but I'll let the docs team be the real judge of that.

Reply:
I've been writing "We recommend", active voice vs passive.